templates/pioneer-study/esper-study-telemetry.1.parquetmr.txt

```diff
@@ -0,0 +1,66 @@
+message esper-study-telemetry {
+  required group metadata {
+    required int64  Timestamp;
+    required binary pioneerId (UTF8);
+    required binary studyName (UTF8);
+    optional binary reason (UTF8);
+    optional binary Date  (UTF8);
+    optional binary DNT (UTF8);
+    optional binary appUpdateChannel (UTF8);
+    required binary documentId (UTF8);
+    optional binary docType (UTF8);
+    optional binary os (UTF8);
+    optional binary sourceName (UTF8);
+    optional binary normalizedChannel (UTF8);
+    optional binary appVendor (UTF8);
+    optional binary clientId (UTF8);
+    optional binary Host (UTF8);
+    required binary geoCountry (UTF8);
+    required binary geoCity (UTF8);
+    optional binary appVersion (UTF8);
+    required binary appBuildId (UTF8);
+    required binary submissionDate (UTF8);
```

Comment:
`submissionDate` doesn't exist for pioneer messages (which is legacy telemetry decoder behavior), so this line should be removed. That data is now available as the s3 partition `submission_date_s3` as configured elsewhere.